### DISCUSSION TITLE
Unterstützung für tasmota - smartmeter Lesekopf

### DISCUSSION THREAD

**Discussion by DeDouby** [tbnobody/OpenDTU#1069] [Ideas]
Hi,
Praktisch wäre eine Unterstützung für tasmota. Oder einen anderen ir Lesekopf

So könnte man vom Zähler die Daten erhalten.

Beispiel: https://www.heise.de/tests/Ausprobiert-Guenstiger-IR-Lesekopf-fuer-Smart-Meter-mit-Tastmota-Firmware-7065559.html

**Comment by CBFOUR** [comment]
Da schau dich mal um >
https://github.com/helgeerbe/OpenDTU-OnBattery

Edit> Oder wolltest du den Lesekopf direkt am ESP32 Löten, quasi beide Funktionen in eins?
        

Cheers

**Reply by DeDouby** [reply]
Das wäre natürlich das Beste. 2in1. 

Aber mit deinem Link Werd ich mich gleich Mal beschäftigen. Danke
